DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -97,7 +97,7 @@ export class SearchSortFilterAndPaginate {
     SearchSortFilterAndPaginate.scrollToTop();
   }
 
-  private searchSortAndPaginate(searchableData: ISearchableData): void {
+  private searchSortAndPaginate(searchableData: ISearchableData, updateResultCount = true): void {

```

Comment:
That is correct. The Default of true here was implemented so it was easy to see where the change will affect. If the value is true, then nothing will have changed from how the code was previously set out. 

So, we do read out the number of results whenever a user hits next/previous page buttons. This is what how it happened before, and acts consistently with the non-JS behaviour. 

I can see how it might be a bit of a pain though, so with the introduction of this parameter, we could easily prevent JS from updating the result count whenever we know a change in the number will not occur (paginating, updating number of items per page, sorting). Though there won't be any sound notification that things have changed elsewhere on the page.

Reply:
Okay, seems fine for now.